File: frontend/src/app/features/dashboard/requesters/dtos/requester.response.dto.ts
import {RequesterType} from '../enums/requester.enum';
import {BaseResponseDto} from '../../../../core/dtos/base-response.dto';

export interface RequesterResponseDto extends BaseResponseDto{
  firstName: string;
  lastName: string;
  personalIdNumber: string;
  emailAddress: string;
  phoneNumber: string;
  type: RequesterType
}

File: backend/src/main/java/io/hb3nce04/keykeeperapp/feature/requester/model/dto/RequesterResponseDto.java
package io.hb3nce04.keykeeperapp.feature.requester.model.dto;

import io.hb3nce04.keykeeperapp.common.model.dto.BaseDto;
import io.hb3nce04.keykeeperapp.feature.requester.model.enums.RequesterType;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class RequesterResponseDto extends BaseDto {
    private String firstName;

    private String lastName;

    private String personalIdNumber;

    private String emailAddress;

    private String phoneNumber;

    private RequesterType type;
}
